tests/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ function(add_uncrustify_test name lang config rerun_config input result output)
   )
 endfunction()
 
-foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma)
+foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma imported)
```

Comment:
This is confusing if one expects every cpp test to be in the cpp.test file. 
If you really need this new category than it would be good to have a comment in the cpp.test file that explains that there are additional tests inside imported.test.